keyboards/vertex/cycle8/info.json

```diff
@@ -0,0 +1,429 @@
+                {"matrix": [1, 12], "x": 12, "y": 1.5},
+                {"matrix": [1, 13], "x": 13, "y": 1.5},
+                {"matrix": [1, 14], "x": 14, "y": 1.5},
+                {"matrix": [1, 15], "x": 15.25, "y": 1.5},
+                {"matrix": [3, 13], "x": 16.25, "y": 1.5},
+                {"matrix": [3, 14], "x": 17.25, "y": 1.5},
+                {"matrix": [2, 0], "x": 0, "y": 2.5, "w": 1.5},
+                {"matrix": [2, 1], "x": 1.5, "y": 2.5},
+                {"matrix": [2, 2], "x": 2.5, "y": 2.5},
+                {"matrix": [2, 3], "x": 3.5, "y": 2.5},
+                {"matrix": [2, 4], "x": 4.5, "y": 2.5},
+                {"matrix": [2, 5], "x": 5.5, "y": 2.5},
+                {"matrix": [2, 6], "x": 6.5, "y": 2.5},
+                {"matrix": [2, 7], "x": 7.5, "y": 2.5},
+                {"matrix": [2, 8], "x": 8.5, "y": 2.5},
+                {"matrix": [2, 9], "x": 9.5, "y": 2.5},
+                {"matrix": [2, 10], "x": 10.5, "y": 2.5},
+                {"matrix": [2, 11], "x": 11.5, "y": 2.5},
+                {"matrix": [2, 12], "x": 12.5, "y": 2.5},
+                {"matrix": [2, 13], "x": 13.5, "y": 2.5, "w": 1.5},
+                {"matrix": [2, 14], "x": 15.25, "y": 2.5},
+                {"matrix": [2, 15], "x": 16.25, "y": 2.5},
+                {"matrix": [3, 15], "x": 17.25, "y": 2.5},
+                {"matrix": [3, 0], "x": 0, "y": 3.5, "w": 1.75},
+                {"matrix": [3, 1], "x": 1.75, "y": 3.5},
+                {"matrix": [3, 2], "x": 2.75, "y": 3.5},
+                {"matrix": [3, 3], "x": 3.75, "y": 3.5},
+                {"matrix": [3, 4], "x": 4.75, "y": 3.5},
+                {"matrix": [3, 5], "x": 5.75, "y": 3.5},
+                {"matrix": [3, 6], "x": 6.75, "y": 3.5},
+                {"matrix": [3, 7], "x": 7.75, "y": 3.5},
+                {"matrix": [3, 8], "x": 8.75, "y": 3.5},
+                {"matrix": [3, 9], "x": 9.75, "y": 3.5},
+                {"matrix": [3, 10], "x": 10.75, "y": 3.5},
+                {"matrix": [3, 11], "x": 11.75, "y": 3.5},
+                {"matrix": [3, 12], "x": 12.75, "y": 3.5, "w": 2.25},
+                {"matrix": [4, 0], "x": 0, "y": 4.5, "w": 1.25},
+                {"matrix": [4, 1], "x": 1.25, "y": 4.5},
+                {"matrix": [4, 2], "x": 2.25, "y": 4.5},
+                {"matrix": [4, 3], "x": 3.25, "y": 4.5},
+                {"matrix": [4, 4], "x": 4.25, "y": 4.5},
+                {"matrix": [4, 5], "x": 5.25, "y": 4.5},
+                {"matrix": [4, 6], "x": 6.25, "y": 4.5},
+                {"matrix": [4, 7], "x": 7.25, "y": 4.5},
+                {"matrix": [4, 8], "x": 8.25, "y": 4.5},
+                {"matrix": [4, 9], "x": 9.25, "y": 4.5},
+                {"matrix": [4, 10], "x": 10.25, "y": 4.5},
+                {"matrix": [4, 11], "x": 11.25, "y": 4.5},
+                {"matrix": [4, 12], "x": 12.25, "y": 4.5},
+                {"matrix": [4, 13], "x": 13.25, "y": 4.5, "w": 1.75},

```

Comment:
Line's 135-136 look identical to a JIS/ABNT2 right shift (i.e. 1u-1.7.u), though there are other split right shift layouts here where split right shift is 1.75u-1u.  
Is intention for this layout that split Right Shift is 1.75u-1u, not 1u-1.75u?